index.bs

```diff
@@ -1918,6 +1918,33 @@ SHOULD be aborted.
     loses focuses. If a hook is provided, the above paragraph will be updated to include the hook.
     See [WHATWG HTML WG Issue #2711](https://github.com/whatwg/html/issues/2711) for more details. 
 
+## Registration Extensions Client Inputs (typedef <dfn>RegistrationExtensionsClientInputs</dfn>) ## {#iface-registration-extensions-client-inputs}
+
+<xmp class="idl">
+    dictionary RegistrationExtensionsClientInputs {
+    };
+</xmp>
+
+This is a dictionary containing the [=client extension input=] values for zero or more WebAuthn extensions, as defined in [[#extensions]].
+
+
+## Registration Extensions Client Outputs (typedef <dfn>RegistrationExtensionsClientOutputs</dfn>) ## {#iface-registration-extensions-client-outputs}
+
+<xmp class="idl">
+    dictionary RegistrationExtensionsClientOutputs {
+    };
+</xmp>
+
+This is a dictionary containing the [=client extension output=] values for zero or more WebAuthn extensions, as defined in [[#extensions]].
```

Comment:
Delete RegistrationExtensionsClientOutputs.  Use AuthenticationExtensionsClientOutputs instead.

Reply:
Done! sorry for the confusion.